Marlin/language_de.h

```diff
@@ -91,73 +91,73 @@
+#define MSG_RESUMING                        "Druckfortsetzung"
 #define MSG_PRINT_ABORTED                   "Druck abgebrochen"
-#define MSG_NO_MOVE                         "Motoren Eingesch."
-#define MSG_KILLED                          "BEENDET."
-#define MSG_STOPPED                         "ANGEHALTEN."
+#define MSG_NO_MOVE                         "Motoren eingeschaltet"
+#define MSG_KILLED                          "KILLED"
+#define MSG_STOPPED                         "ANGEHALTEN"
 #define MSG_CONTROL_RETRACT                 "Retract mm"
 #define MSG_CONTROL_RETRACT_SWAP            "Wechs. Retract mm"
 #define MSG_CONTROL_RETRACTF                "Retract  V"
-#define MSG_CONTROL_RETRACT_ZLIFT           "Hop mm"
+#define MSG_CONTROL_RETRACT_ZLIFT           "Z-Hop mm"
 #define MSG_CONTROL_RETRACT_RECOVER         "UnRet +mm"
 #define MSG_CONTROL_RETRACT_RECOVER_SWAP    "Wechs. UnRet +mm"
 #define MSG_CONTROL_RETRACT_RECOVERF        "UnRet  V"
-#define MSG_AUTORETRACT                     "AutoRetr."
+#define MSG_AUTORETRACT                     "AutoRetract"
 #define MSG_FILAMENTCHANGE                  "Filament wechseln"
-#define MSG_INIT_SDCARD                     "SDKarte erkennen"// Manually initialize the SD-card via user interface
-#define MSG_CNG_SDCARD                      "SDKarte erkennen"// SD-card changed by user. For machines with no autocarddetect. Both send "M21"
+#define MSG_INIT_SDCARD                     "SD-Karte erkennen"  // Manually initialize the SD-card via user interface
+#define MSG_CNG_SDCARD                      "SD-Karte getauscht" // SD-card changed by user. For machines with no autocarddetect. Both send "M21"
 #define MSG_ZPROBE_OUT                      "Sensor ausserhalb"
-#define MSG_YX_UNHOMED                      "X/Y vor Z homen."
+#define MSG_YX_UNHOMED                      "X/Y vor Z homen!"
 #define MSG_ZPROBE_ZOFFSET                  "Z Offset"
 #define MSG_BABYSTEP_X                      "Babystep X"
 #define MSG_BABYSTEP_Y                      "Babystep Y"
 #define MSG_BABYSTEP_Z                      "Babystep Z"
-#define MSG_ENDSTOP_ABORT                   "Endstop Abbr. Ein"
-#define MSG_HEATING_FAILED_LCD              "Heizen fehlgesch."
-#define MSG_ERR_REDUNDANT_TEMP              "Redund. Temperaturabw."
+#define MSG_ENDSTOP_ABORT                   "Endstop-Abbr. ein"
+#define MSG_HEATING_FAILED_LCD              "Heizen fehlgeschlagen"
+#define MSG_ERR_REDUNDANT_TEMP              "Redund. Temperaturabweichung"
 #define MSG_THERMAL_RUNAWAY                 "Temp. n. erreicht"
 #define MSG_ERR_MAXTEMP                     "Temp. überschritten"
 #define MSG_ERR_MINTEMP                     "Temp. unterschritten"
-#define MSG_ERR_MAXTEMP_BED                 "Temp. Bett überschr."
-#define MSG_ERR_MINTEMP_BED                 "Temp. Bett unterschr."
+#define MSG_ERR_MAXTEMP_BED                 "Betttemp. überschritten"
+#define MSG_ERR_MINTEMP_BED                 "Betttemp. unterschritten"
```

Comment:
Noch mal zu den Fehlermeldungen.
Die beinhalen alle ein 'kill'. Möglicherweise alles in GRO?SCHRIFT?
"Temp." könnte durch das Thermometersymbol  (LCD_STR_THERMOMETER) ersetzt werden.